time_series/time_series_aggregations.qmd

```diff
@@ -216,59 +209,39 @@ gs.run_command("t.rast.list", input="annual_spring_warming")
 ```
 
 ```{python}
-spring_warming = gj.TimeSeriesMap(use_region=True)
+spring_warming = gj.TimeSeriesMap()
 spring_warming.add_raster_series("annual_spring_warming", fill_gaps=False)
-spring_warming.d_legend(color="black", at=(10,40,2,6))
+spring_warming.d_legend(color="black", at=(5, 50, 2, 6))
 spring_warming.show()
 ```
 
-:::{.callout-caution title="Question" collapse="true"}
-How would you obtain the average spring warming over the whole time series?
-
-```{python}
-# Average spring warming
-gs.run_command("t.rast.series",
-               input="annual_spring_warming",
-               output="avg_spring_warming",
-               method="average")
-               
-# Display raster map with interactive class
-spw_map = gj.InteractiveMap(width = 500, use_region=True, tiles="CartoDark")
-spw_map.add_raster("avg_spring_warming")
-spw_map.add_layer_control(position = "bottomright")
-spw_map.show()
-```
-:::
```

Comment:
I would not remove this example. The question is just a trigger for the next topic. It's a different way of introducing a topic. And in any case, it is collapsed so it is up the the reader to unfold it.